monitoring/local_monitoring/bulk_demo/bulk_demo.py

```diff
@@ -95,8 +95,8 @@ def thread_print(*args, **kwargs):
         recommendations_json_arr = []
 
         # Hardcodig to reduce the time it takes to list all
-        exp_count=1
-        counter=0
+        exp_count = 1
+        counter = 0
         if exp_list != "":
             print("✅ Complete!")
             print(f"🔄 List the recommendations for {exp_count} experiments...", end="")
```

Comment:
Please update to fetch the recommendations from the job status output instead of list recommendations API as per the new Bulk API format

Reply:
No change in listRecommendations API

Reply:
I meant now that bulk job status output itself has recommendations, fetch the recommendations from the job output instead of listRecommendations

Reply:
done